oktapam/resource_project_group.go

```diff
@@ -223,12 +224,11 @@ func resourceProjectGroupUpdate(ctx context.Context, d *schema.ResourceData, m i
 
 func resourceProjectGroupDelete(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
 	c := m.(client.OktaPAMClient)
-	project, group, err := parseProjectGroupResourceID(d.Id())
-	if err != nil {
-		return diag.FromErr(err)
-	}
 
-	err = c.DeleteProjectGroup(ctx, project, group)
+	project := d.Get(attributes.ProjectName).(string)
```

Comment:
nt. I like what you did above with this being `projectName` / `groupName`

Reply:
Fixed